serviceLibrary/src/main/java/info/mqtt/android/service/MqttAndroidClient.kt

```diff
@@ -41,6 +48,8 @@ class MqttAndroidClient @JvmOverloads constructor(
     private var persistence: MqttClientPersistence? = null) :
     BroadcastReceiver(), IMqttAsyncClient {
 
+    private val scope = CoroutineScope(Dispatchers.IO)
```

Comment:
I tried it with https://github.com/hannesa2/paho.mqtt.android/pull/391/commits/db77a7c36660dc2bbf86f9c303d110e5f649a627

`private val scope = CoroutineScope(Dispatchers.IO + SupervisorJob())`
but with less success